contributing.md

```diff
@@ -0,0 +1,163 @@
+* Our [test coverage](https://codecov.io/github/beetbox/beets) is somewhat low. You can help out by finding low-coverage modules on Coveralls and adding new tests.
+* There are several ways to improve the tests in general (see [Testing](https://github.com/beetbox/beets/wiki/Testing)) and some places to think about performance optimization (see [Optimization](https://github.com/beetbox/beets/wiki/Optimization)).
+* Not all of our code is up to our coding conventions. In particular, the [API documentation][apidocs] are currently quite sparse. You can help by adding to the docstrings in the code and to the documentation pages themselves.
+
+## Your first contribution
+If this is your first time contributing to an open source project, welcome! If you are confused at all about how to contribute or what to contribute, take a look at [this great tutorial](http://makeapullrequest.com/), or stop by our [gitter chat room](https://gitter.im/beetbox/beets#) if you have any questions. 
+
+We maintain a list of issues we feel are perfect for first-timers tagged "[good first issue](https://github.com/beetbox/beets/labels/good%20first%20issue)". These issues are perfect for anyone new to open source. In fact, we encourage only users who are new to open source to tackle these. Ideally, the goal for these issues is to get users comfortable with contributing to an open source project for the first time. Please, do not hesitate to ask any questions.
+
+## How to Submit Your Work
+
+Do you have a great bug fix, new feature, or documentation expansion you'd like to contribute? Follow these steps to create a GitHub pull request and your code will ship in no time.
+
+1. Fork the beets repository and clone it (see above) to create a workspace.
+2. Make your changes.
+3. Add tests. If you've fixed a bug, write a test to ensure that you've actually fixed it. If there's a new feature or plugin, please contribute tests that show that your code does what it says.
+4. Add documentation. If you've added a new command flag, for example, find the appropriate page under `docs/` where it needs to be listed.
+5. Add a changelog entry to `docs/changelog.rst` near the top of the document.
+6. Run the tests and style checker. The easiest way to run the full battery of tests is to use [tox](https://tox.readthedocs.org/en/latest/). This will run the same set of tests that we run on Travis.
+7. Push to your fork and open a pull request! We'll be in touch soon.
+8. If you add commits to a pull request, please *add a comment* after you push them. (GitHub doesn't notify us when commits are added, so we need your comment so we get an email alert.)
+
+Remember, code contributions have four parts: the code, the tests, the documentation, and the changelog entry. Thank you for contributing!
+
+
+# The Code
+
+The documentation has an [API documentation section][apidocs] that serves as an introduction to beets' design. There's also an [[Architecture]] page on this wiki.
```

Comment:
Both of these pages seem to be dead or non-existing

Reply:
Yes, these have moved to the `dev` section of the docs:
https://beets.readthedocs.io/en/stable/dev/index.html